src/index.js

```diff
@@ -383,7 +397,7 @@ export default class ImageTool {
     this._data.file = file || {};
 
     if (file && file.url) {
-      this.ui.fillImage(file.url);
+      this.imageResolver.resolveUrlByFileData(file.url);
```

Comment:
If `resolveUrlByFileData` will return an url, it would be more straightforward 

```suggestion
      const url = await this.imageResolver.resolveUrlByFileData(file);
      
      this.ui.fillImage(url)
```